docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -214,6 +214,7 @@ public class CounterBean {
 * Passivation and passivating scopes are not supported
 * Interceptor methods on superclasses are not implemented yet
 * `@Interceptors` is not supported
+* Decoration of built-in beans, such as `Event`, is not implemented yet
```

Comment:
I'd rather replace _"is not implemented yet"_ with _"is not supported"._ The current wording implies that it will be supported in the future and that does not have to be the case.

Reply:
Ok, will change the wording.